cockroachcloud/architecture.md

```diff
@@ -0,0 +1,76 @@
+
+### Architecture
+
+Traffic comes in from the public internet and is routed by the cloud provider’s load balancer to a Kubernetes cluster that hosts CockroachDB. However, unlike a regular CockroachDB cluster where the SQL and KV layers run in the same process, our serverless architecture fully decouples the SQL layer from the KV layer so that both layers run in separate processes running in separate Kubernetes pods.
+
+<img src="{{ 'images/cockroachcloud/serverless-diagram.jpeg' | relative_url }}" alt="Perf tuning concepts" style="max-width:100%" />
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="dedicated">
+
+## CockroachCloud Dedicated
+
+If you need a single-tenant cluster with no shared resources, we recommend CockroachCloud Dedicated. CockroachCloud Dedicated supports single and multi-region clusters in Amazon Web Services and Google Cloud Platform.
+
+### Hardware
+
+We use the Kubernetes offerings in AWS and GCP (EKS and GKE respectively) to run CockroachCloud offerings. GCP clusters use [N1 standard](https://cloud.google.com/compute/docs/machine-types#n1_machine_types) machine types and [Persistent Disk storage](https://cloud.google.com/compute/docs/disks#pdspecs). AWS clusters use [M5 instance types](https://aws.amazon.com/ec2/instance-types/m5/#Product_Details) and [Elastic Block Store (EBS)](https://aws.amazon.com/ebs/features/). Each single region cluster has a minimum of three nodes spread across three availability zones (AZ) in a cloud provider region. For multi-region clusters, similarly, nodes are spread across three or more AZs in each region.
+
+### Security and Connection
+
+CockroachCloud Dedicated clusters are single tenant. This means each new cluster gets its own project in GCP or its own account in AWS. No two Dedicated clusters share any resources with each other. Since these clusters are within their own accounts and projects, they are also in a default virtual private cloud (VPC). Users connect to a Dedicated cluster by using a load balancer in front of each region which leads to one connection string per region. Unless you set up [VPC peering](network-authorization.html#vpc-peering) or [AWS PrivateLink](network-authorization.html#aws-privatelink), your cluster will use TLS 1.2 protocol for inter-node and client-node communication.
```

Comment:
```suggestion
CockroachCloud Dedicated clusters are single tenant. This means each new cluster gets its own project in GCP or its own account in AWS. No two Dedicated clusters share any resources with each other. Since these clusters are within their own accounts and projects, they are also in a default virtual private cloud (VPC). Users connect to a Dedicated cluster by using a load balancer in front of each region which leads to one connection string per region. Unless you set up [VPC peering](network-authorization.html#vpc-peering) or [AWS PrivateLink](network-authorization.html#aws-privatelink), your cluster will use TLS 1.2 for encrypting inter-node and client-node communication.
```